key_teleop/key_teleop/key_teleop.py

```diff
@@ -161,6 +168,17 @@ def _get_twist(self, linear, angular):
         twist.angular.z = angular
         return twist
 
+    def _get_twist_stamped(self, linear, angular):
```

Comment:
Typically, `getXYZ` is supposed to fetch a value from an object. Here, you are making a new twist stamped object from the arguments, I suggest renaming
```suggestion
    def _make_twist_stamped(self, linear, angular):
```

Reply:
At the same time, we may as well rename `_get_twist` while at it...